src/marqo/index.py

```diff
@@ -499,35 +501,37 @@ def verbosely_add_docs(i, docs):
                                  f'Allowed update_methods: ["replace", "update"] ')
             total_batch_time = timer() - t0
             num_docs = len(docs)
-            mq_logger.info(
```

Comment:
moving this line into the `if isinstance(res, list)` means that it will only output if there is server batching, but the line was intended to be a report of a client batch.

Reply:
also, maybe this is the only message we want to keep as "info", since it's a summary of a client batch, regardless of whether there's server batching or not.

Reply:
Makes sense, but this is my thinking:

- for the server batching it doesn't make sense to have the Marqo execution time (since it is more informative to have this information per server batch, which we can get from DEBUG), so roundtrip time is enough. 
- For the client batching, we want to have the roundtrip and execution time in the same line. 

That's why I split these two info statements out

Reply:
ok makes sense!